### DISCUSSION TITLE
Tmux alters the theme of the pudb debugger

### DISCUSSION THREAD

**Discussion by j93hahn** [inducer/pudb#551] [Troubleshooting] [answered]
Hi, I have my theme set to monokai-256 on a remote SSH connection to a GPU cluster. It's my favorite theme and it looks like this: 
<img width="1440" alt="Screen Shot 2022-09-02 at 12 10 56 PM" src="https://user-images.githubusercontent.com/73632280/188204657-f01bee20-66f4-4bc3-9721-0eac441a1545.png">

However, when I activate tmux, the theme looks completely different (I'm still using monokai-256):
<img width="1440" alt="Screen Shot 2022-09-02 at 12 14 08 PM" src="https://user-images.githubusercontent.com/73632280/188205094-9b377b35-5246-4030-a50a-2c7abb02193a.png">

I know the only difference between the two images is that in the latter, I'm in a tmux session and in the former, I'm not. So the issue has to do with tmux - do you have any ideas as to why tmux might be causing the themes to look so differently? In my case, it is required to use tmux since I am submitting jobs to the cluster which may take several hours or days. 

I guess an alternative may be to debug my scripts outside of the tmux session and then to only use tmux when I am actually running the scripts. However, I'm still curious to see if there's a solution to this. Thank you!

**Comment by inducer** [comment, marked as answer]
AFAICT, there's not much that pudb can do. FWIW, I use pudb in a tmux all the time, and it's just fine, however it's important to make sure the `$TERM` env var is set appropriately. (For me, it ends up being `tmux-256color`.)

**Reply by j93hahn** [reply]
Thanks, that solved it - it was set to `screen` originally!
